### ISSUE TITLE
melg issue

### ISSUE THREAD

**Issue by bereziat** [heimdali/heimdali#62]
melg crashes on the following basic use:
`$ raz -x 256 -y 256 | cco -f > small.h5`
`$ extg -x 100 -y 100 lena.h5 large.h5`
`$ melg small.h5 large.h5`
`melg: ERROR: /Users/bereziat/NOBACKUP/miniconda/envs/heimdali-dev/include/ITK-4.8/itkExtractImageFilter.hxx:99:`
`itk::ERROR: ExtractImageFilter(0x7f93db10e2a0): Extraction Region not consistent with output image`

**Comment by dfroger** [comment]
The test should be:

```
raz -x 256 -y 256 | cco -f > large_blank.h5
extg -x 100 -y 100 lena_f1.h5 small_lena.h5
melg small_lena.h5 large_blank.h5
```

**Comment by dfroger** [comment]
Fixed in bff25774f9ba3a1b552bcb31d4c172753d0cae7b

**Comment by bereziat** [comment]
ok for me
